content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: DNS Proxying
```

Comment:
I'm not sure if it's really needed (given the warning, below), but so far we always mark alpha features as experimental.

```suggestion
title: DNS Proxying [experimental]
```